actions/linting/action.yml

```diff
@@ -23,7 +23,7 @@ runs:
         path: |-
           ~/.ansible
           ~/.cache/pre-commit
-        key: "\
+        key: "\  # yamllint disable-line rule:quoted-strings
```

Comment:
is this the correct way to fix this? Can the `"` removed here? WIll the comment be part of the key?

Reply:
Yeah, sorry, I worked this out recently. [This approach works](https://github.com/astro-informatics/sleplet/blob/7aadfbd66e80f0d0d86f113ee48c92d4b15a17c8/.github/workflows/test.yml#L52-L56).